src/families/tron/bridge/js.js

```diff
@@ -52,7 +74,16 @@ async function doSignAndBroadcast({
       a.currency,
       transport,
       a.freshAddressPath,
-      preparedTransaction
+      {
+        rawDataHex: preparedTransaction.raw_data_hex,
+        assetName: subAccount
+          ? [
+              tokenList.find(
+                t => t.id.toString() === subAccount.token.id.split("/")[2]
+              ).message
+            ] // TODO: Find a better way to store this data ? where ?
```

Comment:
we should give a try to use the `preload()` and `hydrate()` mecanism to fetch and reload the tokens list. I think TRON could be the first attempt to make this dynamic system work and could later be extended to ERC20.

Unless bug, this system should be ready and plugged on desktop and mobile. happy to discuss more next time.

Reply:
Yes I'm not sure about what u want, but I can give a shot ! 👍

Reply:
it's a bit experimental today and it's maybe not even used in the tool so we'll need to discuss and test it together at some point